docs/source/eda/create_report.rst

```diff
@@ -0,0 +1,97 @@
+
+==================================================================
+`create_report`: generate profile reports from a pandas DataFrame
+==================================================================
+
+.. toctree::
+   :maxdepth: 2
+
+Overview
+========
+
+The goal of `create_report` is to generate profile reports from a pandas DataFrame. `create_report` utilizes the functionalities and formats the plots from `dataprep`. It provides the following information:
+
+1. Overview: detect the types of columns in a dataframe
+2. Variables: variable type, unique values, distint count, missing values
+3. Quantile statistics like minimum value, Q1, median, Q3, maximum, range, interquartile range
+4. Descriptive statistics like mean, mode, standard deviation, sum, median absolute deviation, coefficient of variation, kurtosis, skewness
+5. Text analysis for length, sample and letter
+6. Correlations: highlighting of highly correlated variables, Spearman, Pearson and Kendall matrices
+7. Missing Values: bar chart, heatmap and spectrum of missing values
+
+In the following, we break down the report into different sections to demonstrate each section of the report.
+
+Loading dataset
+===============
+Here we load the `titanic` dataset into a pandas dataframe and use it to demonstrate our functionality::
+
+    import pandas as pd
+    df = pd.read_csv("https://raw.githubusercontent.com/datasciencedojo/datasets/master/titanic.csv")
+
+Generate report via `create_report(df)`
+=======================================
+
+After getting a dataset, we could generate the report by calling `create_report(df)`. The following shows an example::
+
+    from dataprep.eda import create_report
+    create_report(df)
+
```

Comment:
I think maybe it is better to provide a link for the reader to download the generated report before we introduce each section.